roles/sahara/tasks/monitoring.yml

```diff
@@ -0,0 +1,2 @@
+---
+
```

Comment:
This empty file should be removed, or it should be populated with monitoring related tasks.